frontend/src/components/GroupCard/GroupList.tsx

```diff
@@ -0,0 +1,66 @@
+import { StyleSheet } from "react-native";
+import { SimpleGrid } from "react-native-super-grid";
+
+import { useResponsiveLayout } from "@/hooks/useResponsiveLayout";
+import { GroupListResponseDataItem } from "@/api/apiSchemas";
+
+import { GroupCard } from "./GroupCard";
+
+interface GroupListProps {
+  isLoading: boolean;
+  groups?: GroupListResponseDataItem[];
+  groupKey: string;
+}
+
+export const GroupList: React.FC<GroupListProps> = ({
+  isLoading,
+  groups,
+  groupKey,
+}) => {
+  const styles = useStyles();
+
+  const { isMobile } = useResponsiveLayout();
+
+  const numColumns = isMobile ? 2 : 4;
```

Comment:
Can we go from 4, 3, and 2?

Reply:
we can, we just need to determine a middle breakpoint. We can create a separate story to add this

Reply:
Sure. Could you create a separate story for this?